pkgs/development/python-modules/beancount-periodic/default.nix

```
@@ -0,0 +1,45 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  unittestCheckHook,
+  setuptools,
+  beancount,
+  beangulp,
+  python-dateutil,
+}:
+
+buildPythonPackage rec {
+  pname = "beancount-periodic";
+  version = "0.2.1";
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  src = fetchFromGitHub {
+    owner = "dallaslu";
+    repo = "beancount-periodic";
+    tag = "v${version}";
+    hash = "sha256-XuBDKG/iOS0gyfiwEEPjIckAbnfOKHjYwXW4CmUy8eA=";
+  };
+
```

```suggestion

  build-system = [ setuptools ];

```
please use the new attributes we try to establish